maven-core/src/main/java/org/apache/maven/plugin/internal/AbstractMavenPluginParametersValidator.java

```diff
@@ -94,19 +98,19 @@ protected boolean isIgnoredProperty(String strValue) {
 
     protected abstract String getParameterLogReason(Parameter parameter);
 
-    protected void logParameter(Parameter parameter) {
-        MessageBuilder messageBuilder = MessageUtils.buffer()
-                .warning("Parameter '")
-                .warning(parameter.getName())
-                .warning('\'');
+    protected String formatParameter(Parameter parameter) {
```

Comment:
Attention: The `MessageBuilder` is here on purpose to highlight stuff. Why is it gone?